src/components/Alert/__stories__/Alert.stories.tsx

```diff
@@ -12,133 +16,227 @@ export default {
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
     },
-    {
-        title,
-        message,
-        theme: 'success',
-        view: 'filled',
+};
+
+const themeCases: Array<AlertProps['theme']> = [
+    'normal',
+    'info',
+    'success',
+    'warning',
+    'danger',
+    'utility',
+];
 
-        actions: [{text: right, handler: console.log}],
+export const Theme: Story = {
+    render: (args) => (
+        <Showcase>
+            {themeCases.map((theme, index) => (
+                <ShowcaseItem title={theme} key={index}>
+                    <Alert {...args} theme={theme} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
     },
-    {
-        title,
-        message,
-        theme: 'warning',
-        view: 'filled',
-        layout: 'horizontal',
-        corners: 'square',
-        onClose: () => {},
-        actions: (
-            <Alert.Actions>
-                <Alert.Action view="outlined">{left}</Alert.Action>
-            </Alert.Actions>
-        ),
+};
+
+const iconCases: Array<{caseTitle: React.ReactNode; propValue: AlertProps['icon']}> = [
+    {caseTitle: 'Without icon', propValue: null},
+    {caseTitle: <IconComponent size={16} data={Gear} />, propValue: 'With icon'},
+];
+
+export const Icon: Story = {
+    render: (args) => (
+        <Showcase>
+            {iconCases.map(({caseTitle, propValue}, index) => (
+                <ShowcaseItem title={caseTitle} key={index}>
+                    <Alert {...args} icon={propValue} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
     },
-    {
-        title,
-        message,
-        theme: 'warning',
-        view: 'outlined',
-        layout: 'horizontal',
-        actions: <Alert.Action>{right}</Alert.Action>,
+};
+
+const cornersCases: Array<AlertProps['corners']> = ['rounded', 'square'];
+
+export const Corners: Story = {
+    render: (args) => (
+        <Showcase>
+            {cornersCases.map((corners, index) => (
+                <ShowcaseItem title={corners} key={index}>
+                    <Alert {...args} corners={corners} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
+    },
+};
+
+const viewCases: Array<AlertProps['view']> = ['filled', 'outlined'];
+
+export const View: Story = {
+    render: (args) => (
+        <Showcase>
+            {viewCases.map((view, index) => (
+                <ShowcaseItem title={view} key={index}>
+                    <Alert {...args} view={view} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
+    },
+};
+
+const layoutCases: Array<AlertProps['layout']> = ['vertical', 'horizontal'];
+
+export const Layout: Story = {
```

Comment:
I see no difference between the cases

Reply:
added actions https://github.com/gravity-ui/uikit/pull/1597/commits/dfd9c5b84d03f6e2ba706d6a3721346399050931

![image](https://github.com/gravity-ui/uikit/assets/15855766/35006a61-ddb4-49ea-aaa8-d279a9a95701)